geode-core/src/main/java/org/apache/geode/cache/wan/GatewaySender.java

```diff
@@ -181,6 +181,13 @@
    */
   void start();
 
+  /**
+   * Starts this GatewaySender and discards previous queue content. Once the GatewaySender is
+   * running,
+   * its configuration cannot be changed.
+   */
+  void startWithClean();
```

Comment:
Can we use something more meaningful as the method name?, `startWithCleanQueue`, `startWithEmptyQueue`, something else?.